### PR TITLE
Fix EZP-21586: ContentTypeService::createContentType() does not check fo...

### PR DESCRIPTION
**Pull request by glye**
...r any permissions

Added access check and unit test. Not sure this is the correct way, seems too simple...

https://jira.ez.no/browse/EZP-21586

### CODE REVIEW

File: eZ/Publish/API/Repository/Tests/ContentTypeServiceAuthorizationTest.php [hunk 1]
```diff
@@ -145,6 +145,62 @@ public function testDeleteContentTypeGroupThrowsUnauthorizedException()
     }
 
     /**
+     * Test for the createContentTypeGroup() method.
```
Comment (joaoinacio) [inline eZ/Publish/API/Repository/Tests/ContentTypeServiceAuthorizationTest.php]: Nitpick: the test is not really for `createContentTypeGroup()` method but `createContentType()`

**Comment by andrerom** [comment]
Missing:
- phpdoc change for API (the interface), and the two other implementations (Signal slot and REST client)
- probably need to adjust the unit tests

Besides that and @joaoinacio 's comment, looks good, so +1

<small>I could nitpick that ´!== true´implies it does not support future policy limitations, but it doesn't right now anyway so it is better that it is consistent with the others in this class.</small>

**Comment by glye** [comment]
@andrerom @joaoinacio
Fixed wrong phpdoc "Test for..."
Added missing phpdoc @throws

**Comment by lolautruche** [comment]
+1

**Comment by pspanja** [comment]
+1

**Pull request merged**